app/ide-desktop/lib/dashboard/src/authentication/cognito.ts

```diff
@@ -367,19 +371,49 @@ export interface UserSession {
   readonly email: string
   /** User's access token, used to authenticate the user (e.g., when making API calls). */
   readonly accessToken: string
+  /** User's refresh token, used to refresh the access token when it expires. */
+  readonly refreshToken: string
+  /** URL to refresh the access token. */
+  readonly refreshUrl: string
+  /** Time when the access token will expire, date in ISO format(UTC). */
+  readonly expireAt: string
+  /** The client ID of the user pool. */
```

Comment:
`of the Cognito user pool` maybe?

Reply:
`Cognito app integration client id` to be most specific